api/review_checklist.md

```diff
@@ -0,0 +1,137 @@
+  the values unique?
+- If a field may eventually be repeated but initially there is a desire to
+  cap it to a single value, consider using repeated with a max length of 1,
+  which is easier to relax in the future.
+
+## Deprecations
+- When a field or enum value is deprecated, according to the minor/patch
+  versioning policy this implies a minor version for support removal. Is the
+  work necessary to add support for the newer replacement field acceptable to
+  known xDS clients in this time frame?
+- No deprecations are allowed when an alternative is "not ready yet" in any
+  major known xDS client (Envoy or gRPC at this point). If you are not sure
+  about the current state of a feature in the major known xDS clients, ask
+  one of the API shepherds.
+- Is this deprecated field documented in a way that points to the preferred
+  newer method of configuration?
+
+## Extensibility
+- Is this feature being directly added into the API itself, or is it being
+  introduced via an extension point?
+- If not via an extension point, why not?
+- If no appropriate extension point currently exists, is this a good
+  opportunity to add one?  Can we move some existing "core" functionality
+  into a set of standardized plugins for an extension point?
+- Do we have good documentation showing what to plug into the extension point?
+  (At minimum, it should have a comment pointing to the common protos to
+  be plugged into the extension point.)
+- If an enum is being introduced, should this be a oneof with empty messages
+  for future API growth?
+- When a new field is introduced for a distinct purpose, should this be a
+  message to allow for future API growth?
+
+## Consistency
+- Can the proposed API reuse part or all of other APIs?
+  - Can some other API be refactored to be part of it, or vice versa?
+  - Example: Can it use common types such as matcher or number?
+- Are there similar structures that already exist?
+- Is the naming convention consistent with other APIs?
+
+## Interactions With Other Features
+- Will this feature interact in strange ways with other features, either
+  existing or planned?
+  - For example, if you are defining a new cluster type, how will the
+    new type implement all of the features currently configured via CDS?
+  - If this is a change in the upstream side of the API, will it work properly
+    with LRS load reporting?
+- Will there be combinations of features that won't work properly?  If so,
+  please document each combination that won't work and justify why this is
+  okay.  Is there some other way to structure this feature that would not
+  cause conflicts with other features?
+
+## Genericness
+- Is this an Envoy-specific or proxy-specific feature?  How will it apply to xDS clients other than Envoy (e.g., gRPC)?
+
+## Dependencies
+- Does this feature pull in any new dependencies for clients?
+- Are these dependencies optional or required?
+- Will the dependencies cause problems for any known xDS clients?
```

Comment:
Can you link to https://github.com/envoyproxy/envoy/blob/master/DEPENDENCY_POLICY.md as an `e.g. for Envoy`? As per my earlier comment, we can't make this policy the API policy, but providing an example is probably good.

Reply:
Done.